.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/

```

Comment:
- [ ] Please I would recommend you add the `test.md` file into your `.gitignore` file because you don't need to push it.